cache/cache.go

```diff
@@ -82,6 +82,7 @@ type BlobCache interface {
 type Reader interface {
 	io.ReaderAt
 	Close() error
+	GetReaderAt() io.ReaderAt
```

Comment:
Can we add a comment like `If a blob is backed by a file, it should return *os.File so that it can be used for FUSE passthrough.`?